### PR TITLE
【Hackathon 5th No.32】为 Paddle 新增 tensor_split / hsplit / dsplit API

### PR DESCRIPTION
**Pull request by megemini**
### PR types

Others

### PR changes

Docs

### Description

增加 `tensor_split / hsplit / dsplit / vsplit` 相应的中文文档。

关联 PR：https://github.com/PaddlePaddle/Paddle/pull/58917

请评审 ～

### CODE REVIEW

File: docs/api/paddle/tensor_split_cn.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.. _cn_api_paddle_tensor_split:
+
+tensor_split
+-------------------------------
+
+.. py:function:: paddle.tensor_split(x, num_or_indices, axis=0, name=None)
+
+
+
+将输入 Tensor 沿着轴 ``axis``` 分割成多个子 Tensor，允许进行不等长地分割。
```
Comment (sunzhongkai588) [inline docs/api/paddle/tensor_split_cn.rst]:
```suggestion
将输入 Tensor 沿着轴 ``axis`` 分割成多个子 Tensor，允许进行不等长地分割。
```

File: docs/api/paddle/dsplit_cn.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.. _cn_api_paddle_dsplit:
+
+dsplit
+-------------------------------
+
+.. py:function:: paddle.dsplit(x, num_or_indices, name=None)
+
+
+
+将输入 Tensor 沿着深度轴分割成多个子 Tensor，等价于将 paddle.tensor_split API 的参数 axis 固定为 2。
```
Comment (sunzhongkai588) [inline docs/api/paddle/dsplit_cn.rst]:
```suggestion
将输入 Tensor 沿着深度轴分割成多个子 Tensor，等价于将 :ref:`cn_api_paddle_tensor_split` API 的参数 axis 固定为 2。
```

File: docs/api/paddle/hsplit_cn.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.. _cn_api_paddle_hsplit:
+
+hsplit
+-------------------------------
+
+.. py:function:: paddle.hsplit(x, num_or_indices, name=None)
+
+
+
+将输入 Tensor 沿着水平轴分割成多个子 Tensor。当 x 的维度大于 1 时等价于将 paddle.tensor_split API 的参数 axis 固定为 1，当 x 的维度等于 1 时等价于将 paddle.tensor_split API 的参数 axis 固定为 0。
```
Comment (sunzhongkai588) [inline docs/api/paddle/hsplit_cn.rst]:
```suggestion
将输入 Tensor 沿着水平轴分割成多个子 Tensor。当 x 的维度大于 1 时等价于将 :ref:`cn_api_paddle_tensor_split` API 的参数 axis 固定为 1，当 x 的维度等于 1 时等价于将 paddle.tensor_split API 的参数 axis 固定为 0。
```

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.dsplit.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+### [paddle.Tensor.dsplit](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor_cn.html#dsplit-num_or_indices-name-none)
+
+```python
+paddle.Tensor.dsplit(num_or_indices, name=None)
+```
+
+其中 Paddle 相比 Pytorch 仅参数名不一致，具体如下：
+
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                    |
```
Comment (sunzhongkai588) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.dsplit.md]:
```suggestion
### 参数映射

| PyTorch       | PaddlePaddle | 备注                                                    |
```
加个空行

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.tensor_split.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+### [paddle.tensor_split](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor_cn.html#tensor_split-num_or_indices-axis-0-name-none)
+
+```python
+paddle.tensor_split(num_or_indices, axis=0, name=None)
+```
+
+其中 Paddle 相比 Pytorch 仅参数名不一致，具体如下：
+
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                    |
```
Comment (sunzhongkai588) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.tensor_split.md]:
```suggestion
### 参数映射

| PyTorch       | PaddlePaddle | 备注                                                    |
```

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.vsplit.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+### [paddle.Tensor.vsplit](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor_cn.html#vsplit-num_or_indices-name-none)
+
+```python
+paddle.Tensor.vsplit(num_or_indices, name=None)
+```
+
+其中 Paddle 相比 Pytorch 仅参数名不一致，具体如下：
+
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                    |
```
Comment (sunzhongkai588) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.vsplit.md]:
```suggestion
### 参数映射

| PyTorch       | PaddlePaddle | 备注                                                    |
```

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.dsplit.md [hunk 7]
```diff
@@ -1,24 +1,23 @@
...
+paddle.dsplit(x,
+        num_or_indices,
         name=None)
 ```
 
-Paddle 相比 PyTorch 支持更多其他参数，具体如下：
+其中 Paddle 相比 Pytorch 仅参数名不一致，具体如下：
+
 ### 参数映射
 | PyTorch       | PaddlePaddle | 备注                                                   |
```
Comment (sunzhongkai588) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.dsplit.md]:
```suggestion
### 参数映射

| PyTorch       | PaddlePaddle | 备注                                                   |
```

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.hsplit.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+```python
+paddle.hsplit(x,
+        num_or_indices,
+        name=None)
+```
+
+其中 Paddle 相比 Pytorch 仅参数名不一致，具体如下：
+
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
```
Comment (sunzhongkai588) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.hsplit.md]:
```suggestion
### 参数映射

| PyTorch       | PaddlePaddle | 备注                                                   |
```

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.tensor_split.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+### [paddle.tensor_split](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/tensor_split_cn.html)
+
+```python
+paddle.tensor_split(x, num_or_indices, axis=0, name=None)
+```
+
+其中 Paddle 相比 Pytorch 仅参数名不一致，具体如下：
+
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                    |
```
Comment (sunzhongkai588) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.tensor_split.md]:
```suggestion
### 参数映射

| PyTorch       | PaddlePaddle | 备注                                                    |
```

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.vsplit.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+```python
+paddle.vsplit(x,
+        num_or_indices,
+        name=None)
+```
+
+其中 Paddle 相比 Pytorch 仅参数名不一致，具体如下：
+
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
```
Comment (sunzhongkai588) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.vsplit.md]:
```suggestion
### 参数映射

| PyTorch       | PaddlePaddle | 备注                                                   |
```

**Comment by megemini** [comment]
@sunzhongkai588 两个小地方改了一下，帮忙 review 一下，辛苦拉 ～～～ 🤗🤗🤗

**Review by sunzhongkai588 — APPROVED** [review]
LGTM
